spec/views/pages/help.html.slim_spec.rb

```diff
@@ -6,4 +6,19 @@
 
     render
   end
+
+  it 'does not contain any improperly-escaped HTML' do
+    render
+
+    expect(rendered).to_not include('&lt;')
+    expect(rendered).to_not include('&gt;')
+  end
+
+  it 'is not missing any interpolation keys' do
+    render
+
+    missing_interpolation_keys = rendered.scan(/%\{[^\}]+\}/)
+
+    expect(missing_interpolation_keys).to be_empty
```

Comment:
I am surprised that the page wouldn't render an error if there is a missing interpolation key?